pkg/core/native/util.go

```diff
@@ -64,6 +70,20 @@ func setUint32WithKey(id int32, dao dao.DAO, key []byte, value uint32) error {
 	return dao.PutStorageItem(id, key, si)
 }
 
+func setIntWithKey(id int32, dao dao.DAO, key []byte, value int64) error {
+	si := &state.StorageItem{Value: bigint.ToBytes(big.NewInt(value))}
+	return dao.PutStorageItem(id, key, si)
+}
+
+func getBigIntWithKey(id int32, dao dao.DAO, key []byte) *big.Int {
```

Comment:
`getInt...() int64`? Everywhere you use it you do `.Int64()` and also this interface is supposed to be symmetrical to `setIntWithKey`.

Reply:
Somehow I forgot about `get...`. Fixed.